### PR TITLE
Test issue with PHP tags being added to blade files

### PR DESCRIPTION
**Pull request by jessarcher**
As requested by @samsonasik, this PR adds a fixture for [this issue](https://github.com/rectorphp/rector/issues/6622) with PHP tags being added to `*.blade.php` files.

The problem is, the test passes! For some reason it's not actually replicating the bug which can be seen online at https://getrector.org/demo/1ebf90f4-e5c3-644e-996f-dfd195cf29da

Is there something I'm missing with my test implementation? Or has this issue already been fixed in an unreleased version?

### CODE REVIEW

File: tests/Issues/PhpTagsAddedToBlade/Fixture/php_tags_added_to_blade.blade.php.inc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@extends('layout')
```
Comment (samsonasik) [inline tests/Issues/PhpTagsAddedToBlade/Fixture/php_tags_added_to_blade.blade.php.inc]: the extension should can be normal `.php.inc`, I think the demo in https://getrector.org/demo/1ebf90f4-e5c3-644e-996f-dfd195cf29da is not related with extension

**Comment by samsonasik** [comment]
@jessarcher thank you, could you try the `"rector/rector": "dev-main"` and verify if it is already resolved?

**Comment by jessarcher** [comment]
Good call. I will try that when I'm back at my computer in the morning.

**Comment by jessarcher** [comment]
Hey @samsonasik, I'm afraid that didn't work. I installed `dev-main` (https://github.com/rectorphp/rector/commit/5690256c72ea3e8313296f3f7381a866bf6daa0f) but the problem still exists.

Any ideas what I could try next to narrow down the problem?

**Comment by jessarcher** [comment]
I've been able to narrow it down a little...

The error **doesn't** occur when `isDryRun` is `false`. However this is normally set to `true` in tests.

I've updated the test to replicate the issue by not doing a dry run. This does leave the working directory in a dirty state after running the test though.

**Comment by samsonasik** [comment]
@jessarcher I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/636

**Pull request closed without merging**